src/_premake_init.lua

```diff
@@ -772,6 +772,20 @@
 		kind = "string",
 	}
 
+	api.register {
```

Comment:
Better to place it in the vs file which  calls the `api.register` as msc specific

Reply:
i put it next to the dotnetframework register one